src/ConfigCatClient.ts

```diff
@@ -221,6 +227,36 @@ export class ConfigCatClient implements IConfigCatClient {
             resolve(null);
         });
     }
+
+    getAllValues(callback: (result: SettingKeyValue[]) => void, user?: User): void {        
+        this.getAllValuesAsync(user).then(value => {
+            callback(value);
+        });
+    }
+    
+    getAllValuesAsync(user?: User): Promise<SettingKeyValue[]> {
+        return new Promise(async (resolve) => {
+            const config = await this.configService.getConfig();
+            if (!config || !config.ConfigJSON || !config.ConfigJSON[ConfigFile.FeatureFlags]) {
+                this.options.logger.error("JSONConfig is not present, returning empty array");
```

Comment:
Suggestion for this and other similar error messages:
`config.json is not present in the cache, returning empty array`
or
`getAllValuesAsync(): config.json is not present in the cache, returning empty array`